server/hazard-service/src/main/java/edu/illinois/ncsa/incore/service/hazard/models/hurricane/Hurricane.java

```diff
@@ -35,6 +35,7 @@ public abstract class Hurricane {
     private String description;
     private Date date = new Date();
     private String creator = null;
+    private String owner = null;
```

Comment:
Same here. Can this be abstracted somehow in a base class?

Reply:
Same answer as anove. But I can do that way if it is really needed